tests/controllers/hibp.test.js

```diff
@@ -72,6 +72,40 @@ test("notify POST with breach, subscriber hash prefix and suffixes should call s
 });
 
 
+test("good notify POST with breach, secondary email hash prefix and suffixes should call sendEmail and respond with 200", async () => {
```

Comment:
lots of code duplication in this test, but I want to get it up for review before I refactor it.